Resources/public/js/views/services/ez-navigationhubviewservice.js

```diff
@@ -23,6 +23,24 @@ YUI.add('ez-navigationhubviewservice', function (Y) {
         initializer: function () {
             this.on('*:logOut', this._logOut);
             this.after('*:navigateTo', this._navigateTo);
+
+            /**
+             * Stores the root struct with it's `location` and `content`
+             *
+             * @property _rootStruct
+             * @type {Object}
+             * @protected
+             */
+            this._rootStruct = null;
```

Comment:
after thinking a bit about it, using those protected properties make the test a bit harder because you'll have add the Location and Content model to the dependencies in the test and you'll have to mock the CAPI and the service. Instead, you should add 2 attributes that are declared like:
```
ATTRS: {
    rootStruct: {
        valueFn: function () {
            return {location: new Y.eZ.Location(), content: new Y.eZ.Content()};
        },
    }
    // same for rootMediaStruct
}
```

this way, in the test you can inject some mocks instead of the real models.